### ISSUE TITLE
用語集：適合している代替版の見直し

### ISSUE THREAD

**Issue by momdo** [waic/wcag21#1722]
Derived https://github.com/waic/wcag21/issues/98

https://waic.jp/translations/WCAG21/#dfn-conforming-alternate-version
> 適合している代替版 (conforming alternate version)
> 代替版は異なる技術環境や利用者層に適応するよう提供される。それぞれの版は可能な限り適合されるべきである。適合要件 1 を満たすためには、そのうちの一つの版が完全に適合したものである必要があるだろう。
> 不適合版と同じように自由に利用可能であるかぎり、適合している代替版は、適合宣言の範囲に含まれている必要はなく、同一のウェブサイト上で提供されている必要もない。

https://www.w3.org/TR/2018/REC-WCAG21-20180605/#dfn-conforming-alternate-version
> Alternate versions may be provided to accommodate different technology environments or user groups. Each version should be as conformant as possible. One version would need to be fully conformant in order to meet conformance requirement 1.
> The conforming alternative version does not need to reside within the scope of conformance, or even on the same Web site, as long as it is as freely available as the non-conforming version.

案：
異なる技術環境又は利用者層に対応するために、代替版が提供される場合がある。それぞれの版は、可能な限り適合されるべきである。適合要件 1 を満たすためには、一つの版が完全に適合している必要がある。
適合している代替版は、不適合版と同様に自由に利用可能である限り、適合宣言の範囲内に存在する必要はなく、同じウェブサイト上に存在する必要もない。
